lib/mosaic-objects/src/main/java/org/eclipse/mosaic/lib/objects/road/SimpleRoadPosition.java

```diff
@@ -192,37 +166,30 @@ public boolean equals(Object obj) {
             return false;
         }
 
-        SimpleRoadPosition srp = (SimpleRoadPosition) obj;
+        SimpleRoadPosition other = (SimpleRoadPosition) obj;
         return new EqualsBuilder()
-                .append(this.connection, srp.connection)
-                .append(this.laneIndex, srp.laneIndex)
-                .append(this.laneOffset, srp.laneOffset)
-                .append(this.lateralLanePosition, srp.lateralLanePosition)
-                .append(this.previousNode, srp.previousNode)
-                .append(this.upcomingNode, srp.upcomingNode)
+                .append(this.connection, other.connection)
+                .append(this.laneIndex, other.laneIndex)
+                .append(this.laneOffset, other.laneOffset)
+                .append(this.lateralOffset, other.laneIndex)
```

Comment:
this probably should be `.append(this.lateralOffset, other.lateralOffset)`

Reply:
Eyes like an eagle. 🦅

Reply:
👁️👃👁️